packages/fhir-team-management/src/components/TeamsDetail/index.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { CloseOutlined } from '@ant-design/icons';
+import { Button } from 'antd';
+import { OrganizationDetail } from '../../types';
+import lang from '../../lang';
+
+export interface TeamsDetailProps extends OrganizationDetail {
+  onClose?: Function;
+}
+
+const TeamsDetail = (props: TeamsDetailProps) => {
+  const { name, active, id, practitioners } = props;
+  const filteredPractitioners = practitioners.filter((prac) => prac.active);
+
+  return (
+    <div className="p-4 bg-white">
+      <Button
+        shape="circle"
+        onClick={() => {
+          if (props.onClose) props.onClose();
```

Comment:
nitpick: props?.onClose()

Reply:
cant do it that `props?.onClose` means if there are props then do onClose... but what if the onCloseis undefined we cant do `onClose?()`